app/app.go

```diff
@@ -305,6 +305,8 @@ func NewMXWApp(logger log.Logger, db dbm.DB) *mxwApp {
 	// We need to customized it
 	rpccore.Routes["debug/fee_info"] = rpc.NewRPCFunc(app.FeeInfo, "")
 	rpccore.Routes["debug/kyc_info"] = rpc.NewRPCFunc(app.KYCInfo, "")
+	rpccore.Routes["debug/ft_auth"] = rpc.NewRPCFunc(app.FTAuth, "")
+	rpccore.Routes["debug/nft_auth"] = rpc.NewRPCFunc(app.NFTAuth, "")
 	rpccore.Routes["debug/ft_info"] = rpc.NewRPCFunc(app.FungibleTokenInfo, "symbol")

```

Comment:
Can we merge `ft_auth` and `ft_auth` commands. So if the symbol attributes set it returns the ft>symbol>info, otherwise ft>info

Reply:
i think its better to follow the convention, ft_info and nft_info. Additionally merge the ft_auth and ft_info into ft_info.

Reply:
The return type is not same. For now we use separate methods.